docker-entrypoint.sh

```diff
@@ -26,4 +26,4 @@ file_env 'YALS_DB_PASSWORD'
 file_env 'TELEGRAM_TOKEN'
 file_env 'BUGSNAG_TOKEN'
 
-exec java ${JAVA_OPTS} -Djava.security.egd=file:/dev/./urandom -jar /app/yals.jar
+exec java ${JAVA_OPTS} -Djava.security.egd=file:/dev/./urandom --add-opens java.base/java.lang=ALL-UNNAMED -XX:+UseContainerSupport -XX:+AlwaysActAsServerClassMachine -jar /app/yals.jar
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/yadevee/yals/pullRequest?prid=5630015)